runtime/compiler/runtime/Listener.hpp

```diff
@@ -37,12 +38,33 @@
  
     The current implementation does not provide code for nicely terminating the listener thread.
  */
+
+#define OPENJ9_LISTENER_POLL_TIMEOUT 10 // in milliseconds
+
+class BaseCompileDispatcher;
+
 class TR_Listener
    {
 public:
    TR_Listener();
    static TR_Listener* allocate();
    void startListenerThread(J9JavaVM *javaVM);
+   void stop();
+   /**
+      @brief Function called to deal with incoming connection requests
+
+      This function opens a socket, binds it and then waits for incoming connection
```

Comment:
Comment needs to be changed to reflect the new implementation.

Reply:
Done.